tools/export/cdt/__init__.py

```diff
@@ -25,15 +56,31 @@ def generate(self):
             'include_paths': [starting_dot.sub('%s/' % self.project_name, inc) for inc in self.resources.inc_dirs],
             'load_exe': str(self.LOAD_EXE).lower()
         }
-
+        
+        launch_cfgs = {}
+        for launch_name in supported_launches:
+            launch = dict(ctx.items())
```

Comment:
This is an odd way to copy. Could you please use the more direct:
```suggestion
            launch = deepcopy(ctx)
```

You will  have to `from copy import deepcopy` at the top of the file.